modules/multibid/index.md

```diff
@@ -0,0 +1,37 @@
+# Overview
+
+Module Name: multibid
+
+Purpose: To expand the number of key value pairs going to the ad server in the normal Prebid way by establishing the concept of a "dynamic alias" -- a bidder code that exists only on the response, not in the adunit. 
+
+
+# Description
+Allowing a single bidder to multi-bid into an auction has several use cases:
+
+1. allows a bidder to provide both outstream and banner
+2. supports the video VAST fallback scenario
+3. allows one bid to be blocked in the ad server and the second one still considered
+4. add extra high-value bids to the cache for future refreshes
+
+
+# Example of using config
+```
+    pbjs.setConfig({
+        multibid: [{
+            bidder: "bidderA",
+            maxbids: 3,
+            targetbiddercodeprefix: "bidA"
```

Comment:
Please ensure correct formatting is listed here (in case anyone tries to use these examples); it seems the code uses `targetBiddercodePrefix`.

Reply:
I'll fix this. I originally coded it all lowercased, and then updated to camelCase. Must have missed this one!